draft-irtf-cfrg-hpke.md

```diff
@@ -1205,6 +1205,23 @@ interface; see {{hpke-export}} for more details.
 
 # API Considerations {#api-considerations}
 
+## Auxiliary Application Information
+
+HPKE has two places at which applications can specify auxiliary authenticated information:
+(1) during context construction via the Setup `info` parameter, and (2) during Context
+operations, i.e., with the `aad` parameter for `Open()` and `Seal()`, and the `info` parameter
+for `Export()`. Application information applicable to multiple operations on a single Context
+should use the Setup `info` parameter. This avoids redundantly processing this information for
+each Context operation. In contrast, application information that varies on a per-message basis
```

Comment:
In addition to redundancy, domain separation is a bit clearer. In ECH, we only include "tls ech" in the `info` parameter. `aad` is a plain ClientHelloOuterAAD. If another protocol decided to put the label in the other fields, injectivity is a little trickier.

Reply:
Yeah, I don't really have a better way to say "domain separation via the key schedule" is preferred without just making this text less clear.